src/components/DataTable/DataTable.tsx

```diff
@@ -317,7 +319,7 @@ class DataTable extends React.PureComponent<CombinedProps, DataTableState> {
 
   private defaultRenderRow = (row: TableData[], index: number) => {
     const className = classNames(styles.TableRow);
-    const {columnContentTypes, truncate = false} = this.props;
+    const {columnContentTypes, truncate = false, alignment} = this.props;
```

Comment:
```suggestion
    const {columnContentTypes, truncate = false, alignment = top} = this.props;
```

Let's either give this a default value, or in cell only give it an alignment class if `alignment` is set.

Reply:
The default value is currently set by `Cell`, since it was providing this behaviour before. Do you think we should move it up to `DataTable`? In that case, would it be better if the prop was required? (Iit doesn't seem like we do that?)